android/lib/network/event/post_event_service.dart

```diff
@@ -0,0 +1,55 @@
+import 'dart:convert';
+
+import 'package:android/config/api_endpoints.dart';
+import 'package:android/network/event/post_event_input.dart';
+import 'package:android/network/event/post_event_output.dart';
+import 'package:flutter/foundation.dart';
+import 'package:http/http.dart';
+
+Future<PostEventOutput> postOnlineEventNetwork(
+    PostOnlineEventInput postEventInput) async {
+  Response response;
+  try {
+    response = await post(
+      Uri.parse("$eventURL/online"),
+      headers: {'Content-Type': 'application/json; charset=UTF-8'},
```

Comment:
the backend side of posting an event will probably require a token in the future. maybe we can add the authentication token to the header already now since it will not change the process.

Reply:
I add the auth token to the post methods headers